Signed-off-by: Robin H. Johnson <robbat2@gentoo.org>
Date: Sat, 24 Feb 2007 18:43:09 -0800

Include a link to the upsset page from the stats page.

--- nut-2.0.5.orig/conf/upsstats.html.sample	2006-11-07 18:08:45.000000000 -0800
+++ nut-2.0.5/conf/upsstats.html.sample	2007-02-24 18:41:06.000000000 -0800
@@ -177,6 +177,7 @@
 
 </td></tr>
 </table>
+<p style="text-align: center"><a href="upsset.cgi">Settings &amp; Commands</a></p>
 
 <hr /><div><small>
 <a href="http://jigsaw.w3.org/css-validator/check/referer"><img style="float:right" src="http://jigsaw.w3.org/css-validator/images/vcss" alt="Valid CSS!" height="31" width="88"/></a>
